docs: fix docker commands (#489)
zwave-js/zwavejs2mqtt: docker/README.md

1. Replace `/dev/ttyACM0` with your serial device
2. Add `-e TZ=Europe/Stockholm` to the `docker run` command to set the correct timezone in container

> 3. Replace `3000:3000` with the Server Port of  the websocket

Reviewer: ```suggestion
3. If you are using zwave-js WS server, replace `3000:3000` with the port choosen in settings
```

### Run using volumes

```bash

> docker run --rm -it -p 8091:8091 -p3000:3000 --device=/dev/ttyACM0 --mount source=zwavejs2mqtt,target=/usr/src/app/store zwavejs/zwavejs2mqtt:latest

Reviewer: ```suggestion
docker run --rm -it -p 8091:8091 -p 3000:3000 --device=/dev/ttyACM0 --mount source=zwavejs2mqtt,target=/usr/src/app/store zwavejs/zwavejs2mqtt:latest
```

```bash
mkdir store

> docker run --rm -it -p 8091:8091 p3000:3000 --device=/dev/ttyACM0 -v $(pwd)/store:/usr/src/app/store zwavejs/zwavejs2mqtt:latest

Reviewer: ```suggestion
docker run --rm -it -p 8091:8091 -p 3000:3000 --device=/dev/ttyACM0 -v $(pwd)/store:/usr/src/app/store zwavejs/zwavejs2mqtt:latest
```
